scio-extra/src/it/scala/com/spotify/scio/extra/voyager/VoyagerIT.scala

```diff
@@ -49,7 +47,7 @@ class VoyagerIT extends PipelineSpec {
 
       val voyagerReader = sc
         .parallelize(sideData)
-        .asVoyagerSideInput(distanceMeasure, storageType, dim)
+        .asVoyagerSideInput(space, numDimensions, storageDataType = storageDataType)
```

Comment:
feels like we should either use named params for both `numDimensions` and `storageDataType`, or neither?

Reply:
`storageDataType` has to be named as there are other default params in between. Will update with all named arguments for clarity
\

Reply:
we have an overload of `asVoyager` with and without `uri`. Onlu the API without uri as default parameters. Should we change this ?